src/services/kpis/kpis.svc.coffee

```diff
@@ -35,6 +35,21 @@ angular
       )
       deferred.promise
 
+    @getKpisDateRange = ->
+      _self.load().then(->
+        kpisDateRange = _self.getCurrentDashboard().metadata.kpis_date_range
```

Comment:
It's called `hist_parameters` everywhere in the API... Could it be something like:
```
Dashboard:
metadata[:hist_parameters][:kpis]
metadata[:hist_parameters][:widgets]
```